experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -127,15 +119,11 @@ function held_suarez_forcing!(bl, source, state, diffusive, aux, t::Real)
     T_equator = FT(315)
     T_min = FT(200)
     σ_b = FT(7 / 10)
-    λ = longitude(bl.orientation, aux)
-    φ = latitude(bl.orientation, aux)
-    z = altitude(bl.orientation, aux)
-    scale_height = _R_d * T_ref / _grav
-    σ = exp(-z / scale_height)
 
-    # TODO: use
-    #  p = air_pressure(T, ρ)
-    #  σ = p/p0
+    # Held-Suarez forcing
+    φ = latitude(bl.orientation, aux)
+    p = air_pressure(T, ρ, bl.param_set)
+    σ = p / _p0
```

Comment:
Leave a TODO here or change: `σ` really should be defined with instantaneous surface pressure in place of `_p0`, for the calculation of `heigh_factor` in relaxation coefficients `k_v` and `k_T` (but not in equilibrium temperature). See design doc.

Reply:
I'll check.